examdownloader-cli.py

```diff
@@ -32,5 +24,28 @@ def downloadCallback(status, lastfile='', numFiles=0):
 
     ed.getContents(module, username, password, destination, downloadCallback, updateStatus)
 
+
 if __name__ == '__main__':
-    startDownload(sys.argv[1:])
+    parser = argparse.ArgumentParser(description="Downloads exam papers from UCL automatically")
```

Comment:
What's UCL?

Reply:
Nevermind, I changed it to NUS Library Portal to avoid confusion